test/mountain_view/presenter_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+class InheritedPresenter < MountainView::Presenter
+  attributes :title, :description
+  attribute :data, default: []
+
+  def title
+    "Foo#{properties[:title].downcase}"
+  end
+end
+
+class MountainViewPresenterTest < ActiveSupport::TestCase
+  def test_partial
+    presenter = MountainView::Presenter.new("header")
+    assert_equal "header/header", presenter.partial
+  end
+
+  def test_locals
+    properties = {name: 'Foo', title: 'Bar'}
+    presenter = MountainView::Presenter.new('header', properties)
+    assert_equal add_properties(properties), presenter.locals
+  end
+
+  def test_attributes
```

Comment:
`attributes` is a private method, which causes this test (and another one below) to raise an error. In general, private methods should not be tested. 
Is there a reason this is tested? Was `attributes` public before?